sdk/security/keyvault/azkeys/example_test.go

```diff
@@ -153,3 +154,40 @@ func ExampleClient_NewListKeyPropertiesPager() {
 		}
 	}
 }
+
+func toByteSlice(s string) []byte {
+	if len(s)%2 == 1 {
+		s = fmt.Sprintf("0%s", s)
+	}
+	ret, err := hex.DecodeString(s)
+	if err != nil {
+		// TODO: handle error
+	}
+	return ret
+}
+
+func ExampleClient_ImportKey() {
```

Comment:
I'm skeptical of this example. The hard parts of the task are
1. converting from whatever key encoding I have to bytes for Key Vault
1. setting the right `JSONWebKey` fields

Part 2 is already documented well on `JSONWebKey` but part 1 may be tricky. This example doesn't really address it because customers will typically get the key from external storage such as a PEM file (looks like [this](https://github.com/Azure/azure-sdk-for-go/blob/54cdd5a3aa7138994b89ddc2c8ee6beb8b890dc6/sdk/azidentity/testdata/certificate.pem#L1-L28)). They won't have hex strings, they'll have base64 or plain old bytes they must parse to extract the key components.

Reply:
Fair point, won't add the sample